api/test/docker/apisix_config2.yaml

```diff
@@ -42,6 +42,7 @@ apisix:
 
 nginx_config:
   error_log_level: "debug"
+  worker_processes: "1"
```

Comment:
1 is the default value for worker_processes, why we need to add it? 🤔

Reply:
default is 'auto'

Reply:
fine, I find it https://github.com/apache/apisix/blob/master/conf/config-default.yaml#L126